MultiTarget/MultiTargetIdentifiers.props

```diff
@@ -46,17 +46,17 @@
     <IsDroid Condition="'$(IsDroid)' == '' AND '$(TargetFramework)' == '$(AndroidLibTargetFramework)' AND '$(AndroidLibTargetFramework)' != '' AND '$(MultiTargetsDroid)' == 'true'">true</IsDroid>
     <IsMacOS Condition="'$(IsMacOS)' == '' AND '$(TargetFramework)' == '$(MacOSLibTargetFramework)' AND '$(MacOSLibTargetFramework)' != '' AND '$(MultiTargetsMacOS)' == 'true'">true</IsMacOS>
     <IsiOS Condition="'$(IsiOS)' == '' AND '$(TargetFramework)' == '$(iOSLibTargetFramework)' AND '$(iOSLibTargetFramework)' != '' AND '$(MultiTargetsiOS)' == 'true'">true</IsiOS>
-    <IsNetstandard Condition="'$(IsNetstandard)' == '' AND $(MultiTargetsNetstandard) == 'true'">true</IsNetstandard>
+    <IsNetstandard Condition="'$(IsNetstandard)' == '' AND '$(TargetFramework)' == '$(NetStandardCommonTargetFramework)' AND '$(MultiTargetsNetstandard)' == 'true'">true</IsNetstandard>
```

Comment:
Can you explain this change? Seems unrelated to WinUI? Is this needed to make that work? 🤔

Reply:
Not needed to make this work, just a bonus hotfix since we saw this broken when using in AppServices